doc/tutorials/setup_openstack_libvirt.rst

```diff
@@ -40,17 +38,15 @@ Install ACRN
    <https://maslosoft.com/kb/how-to-clean-old-snaps/>`_ to clean up old
    snap revisions if you're running out of loop devices.
 #. Make sure the networking bridge ``acrn-br0`` is created. If not,
-   create it using the instructions in XXX.
-
-   .. important:: need instructions from deleted document (using Ubuntu
-      as SOS)
+   create it using the instructions in
+   :ref:`Build and Install ACRN on Ubuntu <build-and-install-acrn-on-ubuntu>`.
 
 Set up and launch LXC/LXD
 *************************
 
 1. Set up the LXC/LXD Linux container engine using these `instructions
    <https://ubuntu.com/tutorials/tutorial-setting-up-lxd-1604>`_ provided
-   by Ubuntu (for release 16.04).
+   by Ubuntu.
```

Comment:
I have followed similar instructions (on Ubuntu 20.04). A couple of things to note:
* I also had to install `lxc-utils` (for the `lxc-checkconfig` utility)
* That tool reports we have some kernel config missing (see below), **but** I believe this is because we use a recent kernel that change this config name to `CONFIG_IP_NF_NAT`. I'm nto done with my testing yet but if true, this is an error the user can actually ignore.

```
CONFIG_NF_NAT_IPV4: missing
CONFIG_NF_NAT_IPV6: missing
```

Reply:
- let me add lxc-utils (for the lxc-checkconfig utility) in the doc. 

- the second error could be ignored from @peterfang 's comments.